lib/ansible/plugins/inventory/script.py

```diff
@@ -23,14 +23,134 @@
         - The source provided must be an executable that returns Ansible inventory JSON
         - The source must accept C(--list) and C(--host <hostname>) as arguments.
           C(--host) will only be used if no C(_meta) key is present.
-          This is a performance optimization as the script would be called per host otherwise.
+          This is a performance optimization as the script would be called one additional time per host otherwise.
     notes:
         - Enabled in configuration by default.
         - The plugin does not cache results because external inventory scripts are responsible for their own caching.
         - To write your own inventory script see (R(Developing dynamic inventory,developing_inventory) from the documentation site.
         - To find the scripts that used to be part of the code release, go to U(https://github.com/ansible-community/contrib-scripts/).
 '''
 
+EXAMPLES = r'''#fmt: code
```

Comment:
The space after `#` is important (https://github.com/ansible/ansible/blob/6ebefaceb6cd0d4961776a94d63a71fc1fc28bc0/test/lib/ansible_test/_util/controller/sanity/yamllint/yamllinter.py#L173):
```suggestion
EXAMPLES = r'''# fmt: code
```